src/vs/workbench/browser/actions/layoutActions.ts

```diff
@@ -1138,6 +1138,37 @@ registerAction2(class extends Action2 {
 	}
 });
 
+// Toggle problem decoration visibility (squiggles)
+registerAction2(class extends Action2 {
```

Comment:
This should move into the markers contribution.

Reply:
this was intended for the command palette - since it's just a global setting now we can just remove this entirely.